package.json

```diff
@@ -39,9 +39,10 @@
   "scripts": {
     "bump": "beachball bump",
     "change": "beachball change",
-    "check": "beachball check",
+    "checkchange": "beachball check  --scope \"!packages/*\" --changehint \"Run 'yarn change' to generate a change file\"",
```

Comment:
I'm checking, but I think we'll want to remove the `!` before "packages". I *think* that is saying, "not that folder".

Reply:
Yeah, I think we want to have `\"packages/*\"`: https://github.com/microsoft/beachball/blob/master/src/monorepo/getScopedPackages.ts#L12

Reply:
makes sense